argparse/argparse.py

```diff
@@ -162,7 +168,7 @@ def _parse_args(self, args):
         # parse all args
         parsed_pos = False
         while args or not parsed_pos:
-            if args and args[0].startswith("-") and args[0] != "-" and args[0] != "--":
+            if args and args[0].startswith("-") and args[0] not in ["-", "--"]:
```

Comment:
Good point. However this hunk is going away because I understand you are OK with lines < 100 characters.